playbooks/roles/edxapp/defaults/main.yml

```diff
@@ -406,6 +406,7 @@ EDXAPP_FEATURES: &edxapp_features
   ENTRANCE_EXAMS: true
   FOOTER_DISCLAIMER_TEXT: "{{ EDXAPP_FOOTER_DISCLAIMER_TEXT }}"
   INDIVIDUAL_DUE_DATES: true
+  MAX_ENROLLEES_FOR_METRICS_USING_DB: "{{ EDXAPP_MAX_ENROLLEES_FOR_METRICS_USING_DB }}"
```

Comment:
Ah, so we had already configured platform to read this from JSON, but never actually wrote it out to the JSON file?